src/app/community/malysis/[analysisId]/components/evidences.tsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+
+import {
+  Report_FileEvidence,
+  Report_ProjectEvidence,
+} from "@buf/safedep_api.bufbuild_es/safedep/messages/malysis/v1/report_pb";
```

Comment:
@kunaldevxxx See the auto-generated code for various Types in the report. We are using them both in frontend and backend (server side rendered code).